### PR TITLE
Add example of corporate profile

### PR DESCRIPTION
**Pull request by stevendanna**
This won't work until the git fetcher is merged. It likely also
exhibits other problems related to platform support handling.

Signed-off-by: Steven Danna steve@chef.io

### CODE REVIEW

**Comment by stevendanna** [comment]
This won't possibly work until https://github.com/chef/inspec/pull/1034 is merged. cc @chris-rock

File: examples/corporate-profile/inspec.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+copyright_email: you@example.com
+license: All Rights Reserved
+summary: An InSpec Compliance Profile
+version: 0.1.0
+depends:
+  - name: ssh-hardening
+    supermarket: hardening/ssh-hardening
+  - name: os-hardening
+    url: https://github.com/dev-sec/tests-os-hardening/archive/master.zip
+  - name: ssl-benchmark
```
Comment (stevendanna) [inline examples/corporate-profile/inspec.yml:14]: ssl-benchmark is twice here, revealing an interesting bug, PR incoming for that.

File: examples/corporate-profile/inspec.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+name: corporate-profile
+title: InSpec Profile
+maintainer: The Authors
+copyright: The Authors
+copyright_email: you@example.com
+license: All Rights Reserved
+summary: An InSpec Compliance Profile
+version: 0.1.0
+depends:
```
Comment (chris-rock) [inline examples/corporate-profile/inspec.yml:9]: nice combinations

**Comment by chris-rock** [comment]
Thanks @stevendanna

**Pull request merged**